lib/session.js

```diff
@@ -44,8 +44,8 @@ function SessionStore(namespace, db, sockets) {
 util.inherits(SessionStore, Store);
 exports.SessionStore = SessionStore;
 
-SessionStore.prototype.createUniqueIdentifier = function() {
-  return uuid.create(128);
+SessionStore.prototype.createUniqueIdentifier = function () {
+  return crypto.randomBytes(64).toString('hex');
```

Comment:
64 bytes means a string of length 128 here, which is the same as it was before. I didn't want to change it from what it was.